pkg/elevator/internal/util/util.go

```diff
@@ -76,3 +85,49 @@ func VersionExposeConfigForVersion(
 	}
 	return catalogv1alpha1.VersionExposeConfig{}, false
 }
+
+func FormDefaults(defaultField *runtime.RawExtension) (defaults map[string]interface{}, err error) {
```

Comment:
FromDefaults ?

Couldn't we have just a `FromRawExtension` function?
This code is not specific to defaults.